src/OrbitGl/App.h

```diff
@@ -324,8 +326,8 @@ class OrbitApp final : public DataViewFactory,
   void SetClipboardCallback(ClipboardCallback callback) {
     clipboard_callback_ = std::move(callback);
   }
-  using SecureCopyCallback =
-      std::function<orbit_base::Future<ErrorMessageOr<void>>(std::string_view, std::string_view)>;
+  using SecureCopyCallback = std::function<orbit_base::Future<ErrorMessageOr<void>>(
+      std::string_view, std::string_view, orbit_base::StopToken)>;
```

Comment:
Since you're already touching the callback, would you mind replacing it by a virtual method in MainWindowInterface?

Reply:
done